SettingsFile.cpp

```diff
@@ -105,6 +105,26 @@ void CheckpointPlugin::writeSettingsFile() {
 9| Freeplay Checkpoint
 9| Bugs/Feature Requests: github.com/NitrOP7674 -or- on Discord: https://discord.gg/SPBxrtfrZw
 9| ** Please make sure to read the README first! **
+8|
+6|Rewind Input Method|rewind_axis|Steer@steer&Throttle@throttle&Pitch@pitch&Yaw@yaw&Roll@roll
+6|Rewind Input Matching Axis Bind|matching_axis|None@none&Steer@steer&Throttle@throttle&Pitch@pitch&Yaw@yaw&Roll@roll
```

Comment:
Maybe a text line after this one that says "Used to prevent accidentally leaving rewind mode due to slight input on the other axis" or something.